suite/tests/api/dis-a64.txt

```diff
@@ -1782,6 +1782,29 @@ fd7fffff : ldr    d31, [sp,#32760]        : ldr    +0x7ff8(%sp)[8byte] -> %d31
 4e7bcc8c : fmla v12.2d, v4.2d, v27.2d : fmla   %q12 %q4 %q27 $0x03 -> %q12
 0e3bcc8c : fmla v12.2s, v4.2s, v27.2s : fmla   %d12 %d4 %d27 $0x02 -> %d12
 
+# FMLA (by element)
```

Comment:
Very trivial nit: Why the multiple spaces in these lines?

Reply:
Removed